nodejs/sample-apps/aws-sdk/deploy/wrapper/main.tf

```diff
@@ -1,6 +1,13 @@
+locals {
+  collector_layer_arn = "arn:aws:lambda:${data.aws_region.current.name}:${var.account_id}:layer:opentelemetry-collector-arm64-0_15_0:1"
```

Comment:
Not a blocker, but should we use `var.architecture` here for the collector?

Reply:
actually can't do that as the layer name is `amd64`, but the architecture enum is `x86_64`